zoo-css-challenge/index.html

```diff
@@ -81,8 +84,8 @@ <h3>Height and Size</h3>
 		<section class="news">
 			<h2>News</h2>
 			<div class="card">
-				<time>2010-04-16</time>
-				<h4>Helping wild turtles in Seychelles</h4>
+				<time>2020-06-36</time>
+				<h4>Which big cat are you?</h4>
```

Comment:
The task was to add a new card, so we should've added a new `div` instead of modifying an existing one